mantle/cmd/kola/qemuexec.go

```diff
@@ -142,21 +139,22 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		configv, _, err := v3.Parse(buf)
+		config, err = conf.Ignition(string(buf)).Render("", false)
```

Comment:
For the calls to `conf.Render` I think we need to pass in the value of `kola.Options.IgnitionVersion == "v2"` (it'd also be good to make a helper to just nab this value rather than having that beefy expression everywhere) rather than a hard-coded value otherwise won't it not be translated to the specified Ignition version?

Reply:
Good catch. I switched the calls I could find and will do a helper.